lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -114,3 +124,40 @@ func ProductVersion() string {
 func VchConfig() *config.VirtualContainerHostConfigSpec {
 	return vchConfig
 }
+
+func createImageStore() error {
+
+	// TODO(jzt): we should move this to a utility package or something
+	host, err := sys.UUID()
+	if host == "" {
+		host, err = os.Hostname()
+	}
```

Comment:
^^^ unless you run the persona on a non nested env.  I don't know how you prefer to develop.  Your call if you want to support running this daemon on a non-nested setup.